src/modules/storage.js

```diff
@@ -1,44 +1,52 @@
 const listsList = document.querySelector('.today-lists');
+// const moveDelete = document.querySelector('.single-list-action-button');
+// const listValue = document.querySelector('.single-list-input');
 
-let listArray = [];
+export let listArray = []; // eslint-disable-line
 
-const addList = () => {
+/* eslint-disable no-unused-vars */
+
+export const addList = () => {
   let listCode = '';
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.